src/netex_france/exporter.rs

```diff
@@ -120,6 +120,13 @@ fn only_alphanumeric(s: &str) -> String {
     s.chars().filter(|c| c.is_alphanumeric()).collect()
 }
 
+fn write_file(filepath: &PathBuf, netex: Element) -> Result<()> {
+    let file = File::create(filepath)?;
+    let mut writer = minidom::quick_xml::Writer::new_with_indent(file, '\t' as u8, 1);
```

Comment:
I would also propose the following modification here to avoid indented files in release mode.

```suggestion
    #[cfg(debug_assertions)]
    let mut writer = minidom::quick_xml::Writer::new_with_indent(file, '\t' as u8, 1);
    #[cfg(not(debug_assertions))]
    let mut writer = minidom::quick_xml::Writer::new(file);
```